tests/functions/test_buffer.py

```diff
@@ -83,17 +83,17 @@ def __call__(self):
         expected_val = [[24, 12.0, 46], [17, 12.0, 29], [10, 11, 12]]
         for v_v, v_e in zip(val, expected_val):
             for v, e in zip(v_v, v_e):
```

Comment:
Can the zip iterations be replaced by a single np.testing.assert_allclose?

Reply:
Fixing